lms/djangoapps/program_enrollments/api/v1/views.py

```diff
@@ -705,3 +714,263 @@ def modify_learner_enrollment_status(self, enrollment_request, program_enrollmen
+                            "url": "https://courses.edx.org/courses/course-v1:edX+AnimalsX+Aardvarks/jump_to/block-v1:edX+AnimalsX+Aardvarks+type@sequential+block@edx_introduction",
+                            "date": "2017-03-05T00:00:00Z"
+                        }
+                    ],
+                    "micromasters_title": "Animals",
+                    "certificate_download_url": "https://courses.edx.org/certificates/123"
+                },
+                {
+                    "course_run_id": "edX+AnimalsX+Baboons",
+                    "display_name": "Breathtaking Baboons",
+                    "course_run_url": "https://courses.edx.org/courses/course-v1:edX+AnimalsX+Baboons/course/",
+                    "start_date": "2018-02-05T05:00:00Z",
+                    "end_date": null,
+                    "course_status": "in-progress"
+                    "emails_enabled": false,
+                    "due_dates": [],
+                    "micromasters_title": "Animals",
+                    "certificate_download_url": "https://courses.edx.org/certificates/123",
+                    "resume_course_run_url": "https://courses.edx.org/courses/course-v1:edX+AnimalsX+Baboons/jump_to/block-v1:edX+AnimalsX+Baboons+type@sequential+block@edx_introduction"
+                }
+            ]
+        }
+    """
+    authentication_classes = (
+        JwtAuthentication,
+        OAuth2AuthenticationAllowInactiveUser,
+        SessionAuthenticationAllowInactiveUser,
+    )
+    permission_classes = (permissions.JWT_RESTRICTED_APPLICATION_OR_USER_ACCESS,)
+
+    @verify_program_exists
+    def get(self, request, program_uuid=None):
+        """
+        Defines the GET endpoint for overviews of course enrollments
+        for a user as part of a program.
+        """
+        user = request.user
+
+        try:
+            user_program_enrollment = ProgramEnrollment.objects.get(
+                program_uuid=program_uuid,
+                user=user
+            )
+        except ProgramEnrollment.DoesNotExist:
+            # if the user is not enrolled in the program, they are not authorized
+            # to view the information returned by this endpoint
+            raise PermissionDenied
+
+        user_program_course_enrollments = ProgramCourseEnrollment.objects.filter(
+            program_enrollment=user_program_enrollment
+        ).select_related('course_enrollment')
+
+        user_program_course_enrollments_ids = [enrollment.course_key for enrollment in user_program_course_enrollments]
+        overviews = CourseOverview.get_from_ids_if_exists(user_program_course_enrollments_ids)
+
+        resume_course_run_urls = get_resume_urls_for_enrollments(user, [enrollment.course_enrollment for enrollment in user_program_course_enrollments])
+
+        response = {
+            'course_runs': [],
+        }
+
+        store = modulestore()
+
+        for index, enrollment in enumerate(user_program_course_enrollments):
```

Comment:
Since you moved it, might as well change the signature. 

If it returned a `collections.OrderedDict` of {course_key: url}, then the callers can change to be `get_resume_urls_for_enrollments(user, enrollments).values()`